lib/pymedphys/tests/dicom/test_coords.py

```diff
@@ -42,54 +42,62 @@ def get_data_file(orientation_key):
     )
 
 
-def run_xyz_function_tests(coord_system):
+def test_axis_extraction_all_orients_and_coord_systems():
     r"""Run the xyz extraction test sequence for a given
     xyz extraction function"""
 
-    expected_xyz_path = download.get_file_within_data_zip(
-        "dicom_dose_test_data.zip", "expected_{}_xyz.json".format(coord_system.lower())
-    )
-
-    with open(expected_xyz_path) as fp:
-        expected_xyz = json.load(fp)
```

Comment:
I'd still like to have a check against a baseline with this (maybe just for one coord system though - all others can be checked relative to this by the new code), but I'd need to resave the `json` and am hazy on where it is/how best to update it!